source/agg-pipeline-builder/create-a-view.txt

```diff
@@ -12,5 +12,44 @@ Create a View from Pipeline Results
    :depth: 1
    :class: singlecol
 
-TODO: 
+To quickly access the results of an aggregation pipeline without having to 
+execute an aggregation pipeline, you can create a :ref:`view <compass-views>` 
```

Comment:
(nit)
Possible reword if I'm reading this correctly. Consider something like the following to avoid repetition and [jargon](https://www.mongodb.com/docs/meta/style-guide/terminology/terms-for-global-audience/).
"To quickly access the results of an aggregation pipeline without having to run it, ...